src/plone/app/theming/utils.py

```diff
@@ -43,6 +43,11 @@
 LOGGER = logging.getLogger("plone.app.theming")
 
 
+def is_truthy(value):
+    """Return `true`, if value is truthy."""
+    return value and str(value).lower() in ("1", "y", "yes", "t", "true")
```

Comment:
First of all, I would rather check for "falsish" values.

That would handle cases where somebody set `DIAZO_ALWAYS_CACHE_RULES` to `on` or `enabled`.

On the other end, there is already some code doing something similar:
https://github.com/plone/plone.app.theming/blob/0f5a53f81114d24299763b28580aa07114d43eca/src/plone/app/theming/transform.py#L48

Then I would not add a helper function for this.

But if you really have to, avoid to handling types that are not strings or call it with another name... `is_truthy(2)` will return `False` which is not immediately clear.